targets/frontend/src/components/contributions/answers/Comment.tsx

```diff
@@ -1,15 +1,26 @@
 import { Box } from "@mui/material";
 import * as React from "react";
-import useFormattedDate from "src/hooks/useFormattedDate";
 
-import { Comments as AnswerComments } from "../type";
+import { AnswerStatus, Comments as AnswerComments } from "../type";
+import { fr } from "@codegouvfr/react-dsfr";
+import { statusesMapping } from "../status/data";
+
+const isAnswerComments = (
+  comment: AnswerComments | AnswerStatus
+): comment is AnswerComments =>
+  (comment as AnswerComments).content !== undefined;
+
+const getText = (comment: AnswerComments | AnswerStatus) =>
+  isAnswerComments(comment)
+    ? comment.content
+    : `Changement de statut: ${statusesMapping[comment.status].text}`;
```

Comment:
On ne changeait par le terme pour un truc plus neutre : 
```suggestion
    : `Statut: ${statusesMapping[comment.status].text}`;
```